assets/constants.ts

```diff
@@ -21,7 +21,9 @@ export = {
   // in the URL for topics that are not yet attached to a classroom.
   "DUMMY_CLASSROOM_URL_FRAGMENT": "staging",
```

Comment:
Maybe CLASSROOM_URL_FRAGMENT_FOR_UNATTACHED_TOPICS or similar.

(sorry, bit long, but trying to distinguish from DEFAULT_CLASSROOM_URL_FRAGMENT)